src/jsp/antlr4/PcsLexer.g4

```diff
@@ -26,256 +26,120 @@ lexer grammar PcsLexer;
 package com.cubrid.plcsql.compiler.antlrgen;
 }
 
-A_LETTER:                     A ;
-ALL:                          A L L ;
+@members {
+private int staticSqlParenMatch = -1;
+}
+
+// keywords that starts Static SQL
+WITH:                         W I T H           { staticSqlParenMatch++; mode(STATIC_SQL); };
+SELECT:                       S E L E C T       { staticSqlParenMatch++; mode(STATIC_SQL); };
+INSERT:                       I N S E R T       { staticSqlParenMatch++; mode(STATIC_SQL); };
+UPDATE:                       U P D A T E       { staticSqlParenMatch++; mode(STATIC_SQL); };
+DELETE:                       D E L E T E       { staticSqlParenMatch++; mode(STATIC_SQL); };
+REPLACE:                      R E P L A C E     { staticSqlParenMatch++; mode(STATIC_SQL); };
```

Comment:
Does this PR consider the REPLACE built-in function without assigning expression (:=)? (that we discussed)
```
REPLACE('12345abcdeabcde','abcde','ABCDE');
```

We do not support CUBRID built-in function calls currently, IMO, we don't have to consider the problem in the issue.

Reply:
Yes. It seems OK to handle it together with built-in functions support.
I believe it can be handled by immediately returning to the default lexer mode at a left parenthesis to the right of a REPLACE token.